ee/dtr/index.md

```diff
@@ -6,61 +6,42 @@ keywords: registry, repository, images
+* [Efficiency](efficiency)
+* [Built-in access control](built-in-access-control)
+* [Security scanning](security-scanning)
+* [Image signing](image-signing)
 
-DTR can be installed on-premises, or on a virtual private
-cloud. And with it, you can store your Docker images securely, behind your
-firewall.
+## Image and job management
 
-You can use DTR as part of your continuous integration, and continuous
-delivery processes to build, ship, and run your applications.
+ DTR has a web-based user interface that you ca use to browse images and [audit repository events](/ee/dtr/user/audit-repository-events/). With the UI, you can see which Dockerfile lines produced an image and, if security scanning is enabled, a list of all of the software installed in that image. You can also [audit jobs with the web interface](/ee/dtr/admin/manage-jobs/audit-jobs-via-ui/).
 
-DTR has a web user interface that allows authorized users in your
-organization to browse Docker images and [review repository events](/ee/dtr/user/audit-repository-events/). It even allows you to see what Dockerfile
-lines were used to produce the image and, if security scanning is enabled, to
-see a list of all of the software installed in your images. Additionally, you can now [review and audit jobs on the web interface](/ee/dtr/admin/manage-jobs/audit-jobs-via-ui/).
+DTR can serve as a Continuous Integration and Continuous Delivery (CI/CD) component, in the building, shipping, and running of applications.
 
 ## Availability
 
-DTR is highly available through the use of multiple replicas of all containers
-and metadata such that if a machine fails, DTR continues to operate and can be repaired.
+DTR is highly available through the use of multiple replicas of all containers and metadata. As such, DTR will continue to operate in the event of machine failure, thus allowing for repair.
 
 ## Efficiency
 
-DTR has the ability to [cache images closer to users](admin/configure/deploy-caches/index.md)
-to reduce the amount of bandwidth used when pulling Docker images.
-
-DTR has the ability to [clean up unreferenced manifests and layers](admin/configure/garbage-collection.md).
+DTR is able to reduce the bandwidth used when pulling Docker images by [caching images closer to users](admin/configure/deploy-caches/index.md). In addition, DTR can [clean up unreferenced manifests and layers](admin/configure/garbage-collection.md).
 
 ## Built-in access control
 
-DTR uses the same authentication mechanism as Docker Universal Control Plane.
-Users can be managed manually or synchronized from LDAP or Active Directory. DTR
-uses [Role Based Access Control](admin/manage-users/index.md) (RBAC) to allow you to implement fine-grained
-access control policies for your Docker images.
+As with Universal Control Plane (UCP), DTR uses [Role Based Access Control (RBAC)](admin/manage-users/index.md), which allows you to manage image access, either manually, with LDAP, or with Active Directory.
```

Comment:
If someone has already sat up LDAP with UCP, they may not be happy if they think they have to do it again with DTR.  We could mention here that DTR gets its RBAC user information from UCP.  This is both good and bad, but when a future DTR version can operate without UCP, we can couch this as a feature.